content/client-lib-development-guide/features.textile

```diff
@@ -606,6 +612,13 @@ h4. ClientOptions
 *** @(TO3k3)@ @realtimeHost@ string - for development environments only; allows a non-default Ably Realtime host to be specified
 *** @(TO3k4)@ @port@ integer - for development environments only; allows a non-default Ably non-TLS port to be specified
 *** @(TO3k5)@ @tlsPort@ integer - for development environments only; allows a non-default Ably TLS port to be specified
+** @(TO3l)@ The follow attributes, if set, are used to change the default behaviour of the libary:
+*** @(TO3l1)@ @disconnectedRetryFrequency@ integer - default 15,000 (15s). When the connection is in the @DISCONNECTED@ state, this delay in milliseconds controls how frequently the client library attempts to reconnect automatically
+*** @(TO3l2)@ @suspendedRetryFrequency@ integer - default 30,000 (30s). When the connection is in the @SUSPENDED@ state, this delay in milliseconds controls how frequently the client library attempts to reconnect automatically
+*** @(TO3l3)@ @httpOpenTimeout@ integer - default 4,000 (4s). Timeout for opening the connection, available in the client library if supported by the transport
+*** @(TO3l4)@ @httpRequestTimeout@ integer - default 15,000 (15s). Timeout for any single HTTP request and response
+*** @(TO3l5)@ @httpMaxRetryCount@ integer - default 3. Max number of fallback host retries for HTTP requests that fail due to network issues or server problems
```

Comment:
Does this just mean 'truncate the fallbackHosts list to the first httpMaxRetryCount`?